### ISSUE TITLE
[indent] - falsy error on multiline element

### ISSUE THREAD

**Issue by DerHerrGammler** [yeonjuan/html-eslint#129]
With a html element like this the indent of the attributes are showed as error but the indent there should be correct.

```html
        <script
            defer
            data-domain="example.com"
            src="https://analytics.example.com/js/script.js"
        ></script>
```

**Comment by yeonjuan** [comment]
@DerHerrGammler Thanks for the report! I'll fix it as soon

**Comment by yeonjuan** [comment]
@DerHerrGammler 
It's fixed and published! Thanks!
```
npm install -D @html-eslint/eslint-plugin@latest
```

**Comment by DerHerrGammler** [comment]
Thank you for fixing this fast 👍
